solutions.rb

```diff
@@ -0,0 +1,32 @@
+# Number one
+def mirror(list)
+  #accesses each item in array, starting from last
+  index = -1
+  backwards = []
+  while list[index] #will stop when reaches nil
+    #copies it
+    #saves it in another array
+    backwards << list[index]
+    index -= 1
+  end
+  #puts them together and returns it.
+  mirror = list + backwards
```

Comment:
This looks good - but, FYI I think this is "ruby magic". (The adding of two arrays.)